src/site/content/en/blog/camera-pan-tilt-zoom/index.md

```diff
@@ -106,10 +106,18 @@ try {
 
 To know whether a Chromium-based browser supports PTZ for a camera, go to the
 internal `about://media-internals` page and check out the "Pan-Tilt-Zoom" column
-in the "Video Capture" tab.
+in the "Video Capture" tab: "pan tilt" and "zoom" respectively mean the camera supports
```

Comment:
```suggestion
in the "Video Capture" tab. "Pan tilt" and "zoom" respectively mean the camera supports
```

Reply:
"pan tilt" is actually what will be written in the "Pan-Tilt-Zoom column". See screenshot

Reply:
I misunderstood what was going on, but there's still a problem; this sentence needs a semi-colon, not a colon.